mudlet-mapper.xml

```diff
@@ -5377,17 +5426,29 @@ function speedwalking(event, num)
       not (
         table.contains(gmcp.Room.Info.details, &quot;indoors&quot;) or
         table.contains(gmcp.Room.Info.details, &quot;considered indoors&quot;)
-      )
+      ) and
+			not (getRoomArea(mmp.currentroom) == 49 and mmp.settings.orbashtan) and
+			not (getRoomArea(mmp.currentroom) == 57 and mmp.settings.orbcyrene) and
+      not (getRoomArea(mmp.currentroom) == 51 and mmp.settings.orbeleusis) and
+      not (getRoomArea(mmp.currentroom) == 55 and mmp.settings.orbhashan) and
+      not (getRoomArea(mmp.currentroom) == 44 and mmp.settings.orbmhaldor) and
+      not (getRoomArea(mmp.currentroom) ==368 and mmp.settings.orbtargossas)
```

Comment:
You'll probably want to use the `orbed` function here as this area ID is crowdmap only.